setup/serial_config_info.md

```diff
@@ -1,18 +1,19 @@
-Configuration can be a bit of a pain for using the serial port on the raspberry pi with ubuntu.
+Configuration can be a bit of a pain for using the serial port on the raspberry pi with ubuntu. You can use either for communications, but if you've followed
+the instructions, you most likely will be using `/dev/serial1`.
```

Comment:
It's not great to sneak these updates in this PR, considering the title is about backporting odometry. Need to at least up the title - I don't think it's worth the effort of opening a new PR with all this stuff.

That said, can you state your reasoning for giving both `serial0` and `serial1` as options? Seems like it would be best to just choose one. Ofc, In my xp, `serial1` was the only one I could get to work, https://github.com/nasa-jpl/osr-rover-code/pull/126/commits/76528ada8e2c452675d7dd4f55dd17111949cae4

Reply:
I'm not sure why you say 'sneaking'. They're clearly in the commit log and in the 'files changed' tab. Also this isn't the main branch. 

I consider these changes to be minimal, just clarifying documentation. I was able to work with both devices (not at the same time). If for some reason someone can only get it to work on `serial0`, this tells them that things should work fine. Someone who follows the instructions properly will still end up with `serial1`, and I'm not suggesting they use `serial0`.

Reply:
I mean, the PR says it's about porting odometry over. But it also touches serial config, idle mode, the launch script. Those aren't minimal to my mind. I'd at least list these changes in the description for the PR, to be explicit.